plejd/types/DeviceRegistry.d.ts

```diff
@@ -13,6 +13,7 @@ export interface OutputDevice {
   name: string;
   output: number;
   roomId: string;
+  roomName: string;
```

Comment:
maybe both this and `roomId` should really be `string | undefined` since we don't require it always?